detection_rules/devtools.py

```diff
@@ -138,6 +142,11 @@ def prune_staging_area(target_stack_version: str, dry_run: bool):
     reversions: List[GitChangeEntry] = []
 
     for change in changes:
+        if str(change.path) in exceptions:
```

Comment:
can you print (or return) the exceptions from this to make it easier to add a comment from the github workflow

Reply:
all this command does it move the changes out of the staging area.
we already modify the commit message to say what files were excluded. and if you want to comment, here is where it would happen, where the list of files is already collected:

https://github.com/elastic/detection-rules/blob/809c06ad5fe1f514ea9bc0bb99502dac60a9f174/.github/workflows/backport.yml#L133-L135

Reply:
I'll look into the PR comment next week and make the change there ^.